app/Http/Controllers/GazetteerController.php

```diff
@@ -228,9 +228,7 @@ private function searchDataitems($parameters)
             ->with(['datasource' => function ($q) {
                 $q->select('id', 'name', 'description', 'link');
             }])
-            ->whereIn('datasource_id', $datasourceIDs)
-            ->join('tlcmap.recordtype', 'tlcmap.dataitem.recordtype_id', '=', 'tlcmap.recordtype.id')
-            ->select('tlcmap.dataitem.*', 'tlcmap.recordtype.type as recordtype_type');
+            ->whereIn('datasource_id', $datasourceIDs);

```

Comment:
Can you separate this change along with the one above into a single commit by referencing its related issue number. #63